package.json

```diff
@@ -43,9 +43,10 @@
     "rewire": "^4.0.1"
   },
   "scripts": {
-    "test": "npm run eslint && npm run cover",
-    "cover": "nyc jasmine",
-    "eslint": "eslint --ignore-path .gitignore ."
+    "eslint": "eslint --ignore-path .gitignore .",
+    "test": "npm run eslint && npm run test:coverage",
+    "test:unit": "jasmine --config=spec/support/jasmine.json",
```

Comment:
Calling this script `test:unit` is misleading, as it actually runs unit and integration test. We could either split that up or just drop this script.